src/ai/backend/client/cli/admin/image.py

```diff
@@ -89,11 +89,12 @@ async def rescan_images_impl(registry: str) -> None:
 @image.command()
 @click.argument('alias', type=str)
 @click.argument('target', type=str)
-def alias(alias, target):
+@click.argument('architecture', type=str, default='x86_64')
```

Comment:
This should be optional since we decided to allow making aliases without explicit architecture fields to resolve the alias with multi-architecture images with same tags at runtime, depending on the available agents. cc: @kyujin-cho